pandas/tests/test_generic.py

```diff
@@ -1539,6 +1540,15 @@ def test_to_xarray(self):
                            expected,
                            check_index_type=False)
 
+    def test_deepcopy_empty(self):
```

Comment:
I'm afraid I don't understand your comment here.  Can you point me to a specific example to base the tests on?